packages/browser/src/transports/base.ts

```diff
@@ -19,14 +19,14 @@ export abstract class BaseTransport implements Transport {
   /**
    * @inheritDoc
    */
-  public async sendEvent(_: Event): Promise<Response> {
```

Comment:
Won't this break custom transports that inherit from the `BaseTransport` if someone ever did this? Not sure if it affects the interface internally during compilation.

Reply:
Nope shouldn't but I will double check the `async/await` syntax is just syntactical sugar it still does return a Promise.